homeassistant/components/fido/sensor.py

```diff
@@ -33,27 +37,108 @@
 REQUESTS_TIMEOUT = 15
 MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
 
+
 SENSOR_TYPES = {
```

Comment:
This can be converted from a dict into a tuple now.

Reply:
Done.